airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
I agree that `field_formatter` is generic, but using `hidden_field_formatter` would be misleading as it implies to handle only hidden values since it's very specific, whereas now this function handles three cases:
1. Hidden values
2. Non-hidden values
3. Invalid values

Case 3 is not just error handling, but applies a format (the `Invalid` label) to the value given the failing condition, so it's just another format that this function handles, making `hidden_field_formatter` name incorrect.
I don't think being generic should be an issue here given the scope of this function. FWIW, I think this function could be called `value_formatter` since `field` is too generic as well.